src/alert-converter/alert-converter.service.ts

```diff
@@ -1,20 +1,32 @@
+import { ConfigService } from '@nestjs/config';
 import { LogMessageFormat, LogType } from 'logging-format';
 /**
- * Class for Converting an Alert into a LogMessage. 
- * TODO: Make things more generic
+ * Class for Converting an Alert into a LogMessage.  * 
  */
 export class AlertConverterService {
+    prometheusUrl: string;
+    windowsExporterUrl: string;
+    constructor(private configService: ConfigService) {
+    }
 
     /**
-     * Takes Alert in correct JSON Format and converts into LogMessage Array.
+     * Takes Alert in correct JSON Format and converts into LogMessage Array. 
+     * If the LogType should be included, the description of a rule in rules.yml needs to contain the LogType.
+     * If Values shall be retrievable the rule needs to contain the field 'VALUE' in the description. 
      * 
      * TODO: needs error handling 
+     * 
      * @param alertJSON the Alert in the Format: https://prometheus.io/docs/alerting/latest/configuration/#webhook_config.
      * @returns an  Array of LogMessages.
      */
     public alertToLogMessages(alertJSON): LogMessageFormat[] {
         let messages: LogMessageFormat[] = [];
 
+        this.prometheusUrl = this.configService.get<string>("PROMETHEUS_URL", "http://localhost:9090/");
+        this.windowsExporterUrl = this.configService.get<string>("WINDOWS_EXPORTER_URL", "http://localhost:9182/metrics");
```

Comment:
hardcoded urls sind bööse. 
oder sind die von prometheus und windowsexporter so unabänderlich vorgegeben?

Reply:
Die Urls stehen in der .env Datei , das dahinter ist nur der Standartwert falls die ULR nicht gelesen werden kann.

Reply:
ahhh, oke, dann ist alle gut :)